Backpack/Card/Classes/BPKCard.h

```diff
@@ -32,6 +40,16 @@ NS_SWIFT_NAME(Card) IB_DESIGNABLE @interface BPKCard: UIControl
  */
 @property (nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
 
+/**
+ * The corner-style to apply to the card.
+ */
+@property (nonatomic, assign) IBInspectable BPKCardCornerStyle cornerStyle;
+
+/**
+ * The default corner-style that is applied to the card when none is specified.
+ */
+@property (nonatomic, readonly) BPKCardCornerStyle BPKCardDefaultCornerStyleValue;
```

Comment:
Im not sure exposing this here is the best, but I'm not sure how else to share the value between `BPKCard` and `BPKDividedCard`

Reply:
We can add a `BPKCardCommon.h` that they can both import? It can define the enum etc?

Reply:
Would still be available to consumers, but much less explicitly. Another option is something like

`BPKCardCornerStyle.h` and `BPKCardConstants.h`. `BPKCard.h` can import `BPKCardCornerStyle.h` and boht `BPKCard.m` and `BPKDividedCard.m` can import `BPKCardConstants.h`

Reply:
Does this add much value over just putting both in `BPKCardCornerStyle.h`

Reply:
So @k0nserv it would be consumed like this?
https://github.com/Skyscanner/backpack-ios/pull/119/commits/835363c89b113f10f030b66050beef2ac863e893#diff-fde14f99208763582e451f22844eed79R42